### ISSUE TITLE
add topics

### ISSUE THREAD

**Issue by Beliavsky** [SMKreyche/SMERCURY-T#1]
I suggest adding topics such as `astronomy`, `general-relativity` in the About section at https://github.com/SMKreyche/SMERCURY-T.

**Comment by TobyOnGithub** [comment]
Hi -- I am compiling with latest gfortran as

$gfortran smercury-t.for -o smercury-t.x

and get the following errors. All of them probably have the same root cause. Any idea on how to solve this?

######################

smercury-T.for:6767:38:

 6767 |         call mio_spl (150,string,nsub,lim)
      |                                      1
Error: Actual argument contains too few elements for dummy argument ‘delimit’ (30/200) at (1)

######################

smercury-T.for:6778:38:

 6778 |         call mio_spl (150,string,nsub,lim)
      |                                      1
Error: Actual argument contains too few elements for dummy argument ‘delimit’ (30/200) at (1)

######################

smercury-T.for:6793:38:

 6793 |         call mio_spl (150,string,nsub,lim)
      |                                      1
Error: Actual argument contains too few elements for dummy argument ‘delimit’ (30/200) at (1)

######################

smercury-T.for:6846:38:

 6846 |         call mio_spl (150,string,nsub,lim)
      |                                      1
Error: Actual argument contains too few elements for dummy argument ‘delimit’ (30/200) at (1)

######################

smercury-T.for:6942:38:

 6942 |         call mio_spl (150,string,nsub,lim)
      |                                      1
Error: Actual argument contains too few elements for dummy argument ‘delimit’ (30/200) at (1)

######################

smercury-T.for:6959:40:

 6959 |           call mio_spl (150,string,nsub,lim)
      |                                        1
Error: Actual argument contains too few elements for dummy argument ‘delimit’ (30/200) at (1)

######################

smercury-T.for:6966:38:

 6966 |         call mio_spl (150,string,nsub,lim)
      |                                      1
Error: Actual argument contains too few elements for dummy argument ‘delimit’ (30/200) at (1)

**Comment by saturnaxis** [comment]
> Hi -- I am compiling with latest gfortran as
> 
> $gfortran smercury-t.for -o smercury-t.x
> 

The 'mercury' code was originally developed in Fortran77, where the latest version of gfortran is checking the MIO_SPL function in new way.  A work around is to compile using the following compiler flags '-O3 -ffixed-line-length-none -std=legacy' or

gfortran -O3 -ffixed-line-length-none -std=legacy smercury-T.for -o smercury_T

Note that the '-O3' is for compiler optimization on the CPU, where the '-ffixed-line-length-none' prevents any errors from any lines that exceed the max number of characters from legacy fortran.

-BQ

**Comment by TobyOnGithub** [comment]
Thank you for quick feedback. Looks like those errors now turned into warnings. Usually, I ignore warnings. However, can they be taken care of as well?

And a related question: I see there are mercury6 versions out there that handles double precision more correctly. Is that an issue for smercury-t?

**Comment by TobyOnGithub** [comment]
Hi --- when executing smercury-t the code exits with a complete integration message but also writing

Note: The following floating-point exceptions are signalling: IEEE_OVERFLOW_FLAG IEEE_DENORMAL

message. Should I worry?

**Comment by saturnaxis** [comment]
> Hi --- when executing smercury-t the code exits with a complete integration message but also writing
> 
> Note: The following floating-point exceptions are signalling: IEEE_OVERFLOW_FLAG IEEE_DENORMAL
> 
> message. Should I worry?

I don't think it should be a problem.  You can check it against a test case but use an  integration that implements a secular approach (e.g., VPLanet) to see if they match.

**Comment by TobyOnGithub** [comment]
Cheers -- thank you so much Billy. Did not hear about VPlanet before.
Billy: I sent you a PM 2 - 3 weeks ago. Maybe it went into your spam folder?
